README.md

```diff
@@ -13,6 +13,52 @@ A data type for optional values.
 * [How To: Provide an easier API for `DateTime`](#how-to-provide-an-easier-api-for-datetime)
 * [Reference: `FromRecord _ _`](#reference-fromrecord-_-_)
 
+## Installation
+
+### Spago
+
+```Dhall
+-- Until this package is included in the latest package set,
+-- add the following in the `additions` part of your `packages.dhall` file.
```

Comment:
What if we added `purescript-option` to the package set? It seems like a pretty big ask for someone consuming this package to have to stay in line with whatever changes here. Also, I'm pretty lazy and don't want to keep this README up to date :blush:.

Reply:
Yeah you're right. I'll try to look into this the next coming week. I don't have knowledge yet on how things go into the package-sets. Also, I read somewhere there will be a new purescript registry. Anyway, I'll get back to this.